examples/merkle-patricia-tree.md

```diff
@@ -1,66 +0,0 @@
-# Verify a Merkle Patricia tree
```

Comment:
Do we need alternative example of `MapProof` usage instead of this?

Reply:
Maybe; in general I think it could make sense to organize examples as JS scripts in order to assure that they are up to date at all times. This is how examples work in Rust, at least :upside_down_face: